src/server/controllers/match.controller.js

```diff
@@ -263,6 +276,34 @@ function findConnection(matchOption, projectOption, localField) {
   ]).exec();
 }
 
+function saveFilter(userId, expertise, career) {
+  return new Promise((resolve, reject) => {
+    Filter.update({
+        user_id: userId,
+        isLatest: true,
```

Comment:
I thought about some external growth about these filter history.
If we keep the history, we can gain potential insights from that.
(e.g. What kind of mentor is needed?)